app/src/fdroid/java/org/fossasia/openevent/fragments/OSMapFragment.java

```diff
@@ -42,6 +50,23 @@ public void onCreate(Bundle savedInstanceState) {
     public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
 
         View rootView = inflater.inflate(R.layout.fragment_map, null);
+        Activity activity = getActivity();
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            if (ContextCompat.checkSelfPermission(activity,
+                    Manifest.permission.WRITE_EXTERNAL_STORAGE)
+                    != PackageManager.PERMISSION_GRANTED) {
+                ActivityCompat.requestPermissions(activity,
+                        new String[]{Manifest.permission.WRITE_EXTERNAL_STORAGE},
+                        PERMISSION_REQUEST_WRITE_EXTERNAL_STORAGE_RESULT);
```

Comment:
here , `BitmapFactory` is trying to fetch a file repeatedly which is located in device storage. As soon we provide the permissions, it gets the access and view gets populated after resume of activity.
So,without overriding the `onRequestPermissionsResult()` we can achieve this.